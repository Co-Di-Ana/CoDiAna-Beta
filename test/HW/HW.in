152
65
158
456152
65
158
456152
65
158
456152
65
158
456152
65
158
456152
65
158
456152
65
158
456152
65
158
456152
65
158
456152
65
158
456152
65
158
456152
65
158
456
-1