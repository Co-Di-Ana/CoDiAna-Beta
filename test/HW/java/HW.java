import cls.A;
import java.util.Scanner;


public class HW {
	public HW() {
		System.out.println("..");
		
		System.out.println("Hello world2!");
		new A ();
		new A ();
		new A ();
		
		Scanner sc = new Scanner (System.in);
		int n = sc.nextInt();
		while (n != -1) {
			System.out.println(n);
			n = sc.nextInt();
		}
		
		//while (true);
			
	}
	public static void main(String[] args) {
		new HW();
	}
} 