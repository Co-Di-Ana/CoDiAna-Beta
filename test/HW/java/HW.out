..
Hello world2!
AAA
AAA
AAA
152
65
158
456152
65
158
456152
65
158
456152
65
158
456152
65
158
456152
65
158
456152
65
158
456152
65
158
456152
65
158
456152
65
158
456152
65
158
456152
65
158
456
